src/colors.css

```diff
@@ -55,12 +61,18 @@ html[data-color-scheme="dark"] {
   --color-danger1: #462522;
   --color-danger1-bw-inverted: #fff;
   --color-danger2: #712f2a;
+  --color-danger2-bw-inverted: #fff;
   --color-danger4: #e0584d;
+  --color-danger4-bw-inverted: #fff;
```

Comment:
Should be #000 for danger4-inverted

Reply:
According to the l value, yes. According to my eyes looking at how it looks in the editor, no.

But I am willing to concede that "my eyes" are not a terribly good metric.

Reply:
Hu good point. My eyes agree with yours... huff, tricky. The [contrast checker](https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=E0584D) says that black text has better contrast, agreeing with the `L > 50` rule. Uhm... 

Maybe whats nicer for the eyes also depends on the color scheme or rather the background:

![red-compare](https://github.com/opencast/appkit/assets/7419664/3244990d-afa2-4bb3-a25d-bfde1d097ba0)

In light mode, I think the white text in red is easier to read. But on dark mode... while white text still has lots of contrast to the red, the black text is nicer? (The button background is always `#e0584d`, Tobira around it is just as context)